### DISCUSSION TITLE
Is there a yarn3 version of yarn1's --pure-lockfile option?

### DISCUSSION THREAD

**Discussion by tomskopek** [yarnpkg/berry#4852] [Q&A] [answered]
yarn 1 had the [--pure-lockfile option](https://classic.yarnpkg.com/en/docs/cli/install#toc-yarn-install-pure-lockfile) to not generate a yarn.lock file: `yarn install --pure-lockfile`

We used this in our build script for our server.
Is there any equivalent for yarn3?

**Comment by arcanis** [comment, marked as answer]
No

**Reply by tomskopek** [reply]
Thanks
